script/check-config.sh

```diff
@@ -99,7 +100,8 @@ check_flags() {
 }
 
 check_distro_userns() {
-	source /etc/os-release 2>/dev/null || /bin/true
+	# shellcheck source=/dev/null
+	source /etc/os-release 2>/dev/null || return 0
```

Comment:
Pursuant to https://github.com/moby/moby/pull/43230, I think this probably makes sense to also be a bit more explicitly protective of? :innocent:

```suggestion
	if [ ! -e /etc/os-release ]; then
		return 0
	fi
	# shellcheck source=/dev/null
	source /etc/os-release 2>/dev/null || return 0
```

(Although I see we're still `bash` here, hence `source` vs `.`, so grain of salt)

Reply:
This new check better be `-r` not `-e` since if the file is present but not readable, shell may still error out.

Added.